### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by iAmWaris97**
- Removed all hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task.
- Implemented a function for deleting a task.
- Implemented a function for editing task descriptions.
- By default, new tasks have the property completed set to false.
- Deleting a task updates all remaining items' indexes.
- All changes to the To Do List are saved in local storage.

### CODE REVIEW

File: src/modules/crud.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    const label = document.createElement('input');
+    label.value = `${tasksList[i].description}`;
+    label.classList.add('description');
+    label.setAttribute('type', 'text');
+    if (tasksList[i].completed) {
+      label.style.textDecorationLine = 'line-through';
+    }
+
+    label.setAttribute('readOnly', 'readOnly');
```
Comment (BertrandConxy) [inline src/modules/crud.js:26]: - [x] [OPTIONAL] This file contains a lot of codes and as we have learned how to split our codes into small individual files called modules I would suggest that you try to reduce the codes in this crud function by making small individual functions and then you place them in their individual files. Then in order to use them you know, you can use import/export javascript ES6 features 😉 The only way we master what we learn is by practicing them and you need to start with small projects like this 😎

File: src/modules/crud.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      if (edit.innerHTML === '✏️') {
+        label.removeAttribute('readonly');
+        label.focus();
+        edit.innerHTML = '💾';
+      } else {
+        const edited = label.value;
+        const idNumber = event.target.id;
+        label.setAttribute('readonly', 'readonly');
+        edit.innerHTML = '✏️';
+
```
Comment (BertrandConxy) [inline src/modules/crud.js:78]: - [x] Good job so far 👏 You are almost there 😃. You have implemented almost all functionalities except the delete task functionality. **I can not be able to delete a task.** I have seen that you implemented the functionality to clear or delete all tasks that are completed. Well, **that's great but we have to implement the feature that allows the user to delete or remove a task whenever he/she needs regardless of whether it is completed or not** 😃 so please implement that 🏃 
**NOTE** Remember to keep the remaining tasks' indexes implemented. This is among the project requirements too and kudos for you have implemented it on the clearAllComplete feature 😉

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @iAmWaris97 , 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪

- Best UI 🥇 
- No linter errors
- correct flow is used
- good PR
- good README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/crud.js (hunk 2 above)
Comment (iAmWaris97) [reply src/modules/crud.js:78]: Made the requested changes, and now I have a button to delete any task.

File: src/modules/crud.js (hunk 1 above)
Comment (iAmWaris97) [reply src/modules/crud.js:26]: I will do this in later stages, but for now please consider the way I did. Thank you for the kind review.

**Review by EricMbouwe — APPROVED** [review]
Hi @iAmWaris97, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Linters ✔
- PR ✔
- Requested changes ✔
## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**